tableaudocumentapi/multilookup_dict.py

```diff
@@ -46,11 +49,11 @@ def __setitem__(self, key, value):
 
         dict.__setitem__(self, key, value)
 
-    def get(self, key, default_value=AttributeError):
+    def get(self, key, default_value=_no_default_value):
         try:
             return self[key]
-        except AttributeError:
-            if default_value != AttributeError:
+        except KeyError:
+            if default_value != _no_default_value:
```

Comment:
Since we're checking for that object you can use "is not" for identity checking, and then it's impossible for any overridden __eq__ to trick it